wsagent/che-datasource-api/src/main/java/org/eclipse/che/datasource/api/AvailableDriversService.java

```diff
@@ -0,0 +1,82 @@
+package org.eclipse.che.datasource.api;
```

Comment:
Missing license header. Should probably keep EPL https://github.com/codenvy-legacy/plugin-datasource/blob/master/codenvy-ext-datasource-server/src/main/java/org/eclipse/che/ide/ext/datasource/server/AvailableDriversService.java#L1-L10 
Maybe need to update the years.